data/wanderer/wanderers start.txt

```diff
@@ -1343,8 +1351,8 @@ mission "Wanderers: Alpha Surveillance F"
 			choice
 				`	"Yes."`
 					goto jump
-				`	"Wait, you know about the Hai?"`
-			`	Danforth looks bemused. "Living on their doorstep? Of course I do. Why do you think the Navy maintains such a large base on Farpoint in the first place? It's not like the human anarchists up north pose that much of a threat."`
+				`	"You seem to be very familiar with Hai space?"`
```

Comment:
```suggestion
				`	"You seem to be very familiar with Hai space."`
```

Reply:
Because this sentence isn't framed in the form of a question, it shouldn't have a question mark at the end.